src/pages/index.js

```diff
@@ -119,7 +198,26 @@ profileAddButton.addEventListener("click", () => {
   profileAddPopup.open();
 });
 
-// // Initial render
-cardSection.renderItems();
+profileAvatarButton.addEventListener("click", () => {
+  profileAvatarPopup.open();
+});
 
-// Will add needs corrected in future
+document.addEventListener("DOMContentLoaded", async () => {
+  console.log("DOM fully loaded and parsed");
+  try {
+    const userInfoData = await api.getUserInfo();
+    currentUserId = userInfoData._id;
+    userInfo.setUserInfo(userInfoData);
+
+    const initialCardsData = await api.getInitialCards();
+    initialCardsData.forEach((data) => {
```

Comment:
* You need to use method `renderItems` of class `Section` for rendering the array of cards